### PR TITLE
Proposal to support OpenShift Routes

### PR DESCRIPTION
**Pull request by disposab1e**
Just to let you know what I have done so far...

### CODE REVIEW

File: charts/argo-cd/values.yaml [hunk 1]
```diff
@@ -85,6 +86,14 @@ ingress:
     #   hosts:
     #     - chart-example.local
 
+# Create a OpenShift Route with SSL passthrough for UI and CLI
+# Consider setting 'hostname' e.g. https://argocd.apps-crc.testing/ using your Default Ingress Controller Domain
```
Comment (disposab1e) [inline charts/argo-cd/values.yaml]: Wrong! Example should be: argocd.apps-crc.testing

**Comment by seanson** [comment]
Looking good! Could you please bump the `Chart.yaml` version to 0.7.3 and we'll be ready for publish.

**Comment by seanson** [comment]
Apologies for the delay on this, I've raised a separate PR to handle the chart linting issues here https://github.com/argoproj/argo-helm/pull/146, once that is merged this should be clear to go.

**Comment by disposab1e** [comment]
All easy! I saw it, will wait for the merge and will update my branch of PR.

**Comment by seanson** [comment]
Thanks for your patience! The main two PRs pending for this (1.0.0 rewrite and CI cleanup) have been merged. Please rebase your changes and ensure they're working as expected with the new chart and we should be good to go.

**Comment by disposab1e** [comment]
Close PR and create anew one with tidy history

**Pull request closed without merging**